pvanalytics/quality/time.py

```diff
@@ -81,27 +66,32 @@ def shifts_ruptures(event_times, reference_times, period_min=2,
         timezone. For example, passing solar transit time in a fixed offset
         time zone can be used to detect daylight savings shifts when it is
         unknown whether or not `event_times` is in a fixed offset time zone.
-    period_min : int, default 2
+    period_min : int, default 15
         Minimum number of days between shifts. Must be less than or equal to
         the number of days in `event_times`. [days]
-
         Increasing this parameter will make the result less sensitive to
         transient shifts. For example if your intent is to find and correct
         daylight savings time shifts passing `period_min=60` can give good
         results while excluding shorter periods that appear shifted.
     shift_min : int, default 15
         Minimum shift amount in minutes. All shifts are rounded to a multiple
         of `shift_min`. [minutes]
-    round_up_from : int, optional
-        The number of minutes greater than a multiple of `shift_min` for a
-        shift to be rounded up. If a shift is less than `round_up_from` then
-        it will be rounded towards 0. If not specified then the shift will
-        be rounded up from `shift_min // 2`. Using a larger value will
-        effectively make the shift detection more conservative as small
-        variations will tend to be rounded to zero. [minutes]
     prediction_penalty : int, default 13
         Penalty used in assessing change points.
         See :py:meth:`ruptures.detection.Pelt.predict` for more information.
```

Comment:
I guess this cross-reference should be updated to something with Binseg instead of Pelt

Reply:
Updated to binseg from pelt in https://github.com/pvlib/pvanalytics/pull/197/commits/6da960aea9359c85da39ccfcbba71cea69e83cdc